cli/package/windows/main.wxs

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"
```

Comment:
add xml comment that this file has been generated by command xyz + specify modifications